www/gatsby-config.js

```diff
@@ -87,6 +86,14 @@ module.exports = {
   },
   plugins: [
     `gatsby-plugin-theme-ui`,
+    {
+      resolve: `gatsby-alias-imports`,
+      options: {
+        aliases: {
+          "@components": "src/components",
```

Comment:
Can we add comment explaining reasoning for using this? Something like:
```suggestion
          // We can't use relative imports in translation repositories, so we will use `@components` alias in MDX content instead
          "@components": "src/components",
```
(possibly adding bit more explanation why we can't use relative imports in translation repositories)